routes/index.js

```diff
@@ -1,11 +1 @@
-import express from 'express';
-
-const router = express.Router();
-
-router.get('/', (req, res) => {
-  res.status(200).json({
-    message: 'Hello Books Deferral'
-  });
-});
-
-export default router;
+// file to be used for future endpoints
```

Comment:
Please, could you also clarify why the contents of this file were deleted? My opinion is that the router should still be declared and exported. I don't think it will make our code any worse